TST: Cron jobs for s390x and arm64 failed with hypothesis error

Example log: https://travis-ci.org/github/astropy/astropy/builds/694700963

```
____________ ERROR collecting astropy/time/tests/test_precision.py _____________
astropy/time/tests/test_precision.py:9: in <module>
    from hypothesis import assume, example, given, target
E   ImportError: cannot import name 'target' from 'hypothesis' (/usr/lib/python3/dist-packages/hypothesis/__init__.py)
____________ ERROR collecting astropy/time/tests/test_precision.py _____________
ImportError while importing test module '/home/travis/build/astropy/astropy/astropy/time/tests/test_precision.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
astropy/time/tests/test_precision.py:9: in <module>
    from hypothesis import assume, example, given, target
E   ImportError: cannot import name 'target' from 'hypothesis' (/usr/lib/python3/dist-packages/hypothesis/__init__.py)
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!
```

@mhvk or @Zac-HD , any ideas? Thanks.

Looks like yet another problem with distro-provided packages :roll_eyes:   
(we [explicitly ask](https://hypothesis.readthedocs.io/en/latest/packaging.html) not to be repackaged unless you keep up to date, and yet...)

```
Installing collected packages: ..., hypothesis, ...
    ...
  Attempting uninstall: hypothesis
    Found existing installation: hypothesis 4.36.2
    Not uninstalling hypothesis at /usr/lib/python3/dist-packages, 
        outside environment /home/travis/build/astropy/astropy/tests
    Can't uninstall 'hypothesis'. No files were found to uninstall.
Successfully installed ... hypothesis-5.16.0 ...

============================= test session starts ==============================
plugins: ..., hypothesis-4.36.2, ...
____________ ERROR collecting astropy/time/tests/test_precision.py _____________
astropy/time/tests/test_precision.py:9: in <module>
    from hypothesis import assume, example, given, target
E   ImportError: cannot import name 'target' from 'hypothesis'
        (/usr/lib/python3/dist-packages/hypothesis/__init__.py)
```

In short: `pip install` encountered a much older version of Hypothesis, attempted to replace it with the required new version, and *failed but reported success anyway*, and then the tests failed because they do in fact require the newer features.

You can probably get around this with the `--user` flag, but IMO you'd be much better off with a virtualenv and avoiding dist-packages entirely.

--- a/.travis.yml
+++ b/.travis.yml
@@ -46,7 +46,7 @@ env:
 
         # For installing without tox, we can rely mostly on what Debian already
         # includes.  This is only used if SETUP_METHOD='apt'.
-        - APT_DEPENDENCIES="python3-pip python3-dev python3-venv python3-setuptools cython3 ipython3 python3-jinja2 python3-numpy python3-pytest-astropy python3-pytest-cov python3-pytest-xdist python3-pytest-filter-subpackage python3-objgraph python3-coverage tzdata"
+        - APT_DEPENDENCIES="python3-pip python3-dev python3-venv python3-setuptools cython3 ipython3 python3-jinja2 python3-numpy python3-pytest-astropy python3-pytest-cov python3-pytest-xdist python3-pytest-filter-subpackage python3-objgraph python3-coverage python3-attr tzdata"
 
         # The following is needed to avoid issues if e.g. Matplotlib tries
         # to open a GUI window.

--- a/astropy/table/pandas.py
+++ b/astropy/table/pandas.py
@@ -29,5 +29,5 @@ try:
     display.display(dhtml)
 except ImportError:
     print(ascii_uncoded)
-except UnicodeEncodeError:
+except (UnicodeEncodeError, SyntaxError):
     pass

